### ISSUE TITLE
Unwanted languages downloaded

### ISSUE THREAD

**Issue by PoloDePoli** [Kalanyr/gogrepoc#87]
On my set-up (and ever since) I have updated the manifest through `python gogrepoc update -os windows -lang en`.
When downloading the games via `python gogrepoc download`, other languages beyond english are downloaded

E.g.
1. Master of Magic Classic: french and german were also downloaded
2. The Witcher 3 Wild Hunt Game: 
    - br, de, fr, jp, pl, ru language packs; 
    - bonus packs in french, german, italian, polish, russian & spanish
3. Project Eden: french manual
4. Syndicate: french manual and reference card
5. Forgotten Realms Unlimited Adventures: german manual and reference card
6. The Witcher: italian, polish & german manual and game guides
7. + other like the above examples

Is this the expected behaviour?

**Comment by Kalanyr** [comment]
Extras (eg manuals) don't have languages specificied so they always get
included if extras are allowed. Some of these could get tagged
heuristically but extras don't have well defined naming conventions.

And GOG for some reason doesn't tag language pack patches with their
language, so they just look like patches.  I could probably tag nearly  all
of these heuristically.

Given that:

I would have to check Master of Magic Classic (sometimes languages/OSes
that aren't officially supported get included as extras, so it's probably
correct behaviour).

The others are are as expected.

**Comment by PoloDePoli** [comment]
Thanks - for old games the extra languages are not too bad, however for newer ones (e.g. The Witcher) it can become quite a significant size (~30gb) of unwanted material

**Comment by Kalanyr** [comment]
Yeah, it's not ideal. You should be able to use the -skip flag I think it
is to specify filenames to skip, I *think* it allows wild cards (*?).

**Comment by PoloDePoli** [comment]
So I would use -skip bonus_pack_*.zip *french* *german* *italian* *polish* *russian* building an extensive list of all the files I do not want?

**Comment by Kalanyr** [comment]
It's not that sophisticated unfortunately. Wildcards match all matches and
can't be constrained like that unfortunately. Though english extras usually
skip language tagging in the name  at all so you rarely have to worry about
false positives there.

**Comment by Kalanyr** [comment]
I did just have a look at Master of Magic Classic, and the behaviour with French/German are "correct" in that they are indeed supplied as unsupported extras, so gogrepoc treats them as languageless extras .
